result_after.txt

```diff
@@ -0,0 +1,60 @@
+sct_testing, 169.367
```

Comment:
please don't commit that, the right place is to have the tables in your PR body to explain the benefits of the changes